include/sys/signal.h

```diff
@@ -100,6 +100,7 @@ int sig_check(thread_t *td);
  * If the default action for a signal is to terminate the process and
  * corresponding signal handler is not set, the process calls `sig_exit`.
  *
+ * \note Must be called with all_proc_mtx and proc::p_mtx locked!
```

Comment:
I would clarify that the current process's `p_mtx` must be held.

Reply:
Done!